ratelimiter/ratelimiter.go

```diff
@@ -42,6 +47,15 @@ func resolveBucketStore(config map[string]string) (leakybucket.Storage, error) {
 	case "redis":
 		return leakybucketRedis.New("tcp", fmt.Sprintf("%s:%s",
 			config["host"], config["port"]))
+	case "dynamodb":
+		return leakybucketDynamoDB.New(
+			config["table"],
+			session.New(&aws.Config{
+				Region:     aws.String(config["region"]),
+				MaxRetries: aws.Int(0),
+			}),
+			24*time.Hour,
```

Comment:
Is there any sense in exposing this to the Sphinx configuration? Even if not, I feel like it should maybe be explained or mentioned somewhere, since as far as I can tell nothing else in sphinx is constraining the ratelimit to be less than the scale of days.

Reply:
This is purely for the TTL of the items we're storing in dynamodb: https://godoc.org/github.com/Clever/leakybucket/dynamodb#New

doesn't affect the ratelimiting!

Reply:
And I suspect not. I think it's easy enough to add in the future if we think there's any value in the additional toggle